tfhe/benches/integer/glwe_packing_compression.rs

```diff
@@ -0,0 +1,70 @@
+use criterion::{black_box, criterion_group, Criterion};
+use tfhe::integer::ciphertext::CompressedCiphertextListBuilder;
+use tfhe::integer::{ClientKey, RadixCiphertext};
+use tfhe::shortint::parameters::list_compression::COMP_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
+use tfhe::shortint::parameters::PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
+
+fn glwe_packing(c: &mut Criterion) {
+    let param = PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
+
+    let comp_param = COMP_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64;
+
+    let bench_name = "integer_packing_compression";
+
+    let mut bench_group = c.benchmark_group(bench_name);
+
+    let cks = ClientKey::new(param);
+
+    let private_compression_key = cks.new_compression_private_key(comp_param);
+
+    let (compression_key, decompression_key) =
+        cks.new_compression_decompression_keys(&private_compression_key);
+
+    for num_bits in [8, 16, 32, 64, 128, 256] {
```

Comment:
doest his also cover the max capacity of the GLWE ?

Reply:
Yes, `lwe_per_glwe` is 256

Reply:
except this is a number of bits so we would have 128 bodies, we should have a separate case that builds a number of bits for the max capcity 

like

256, lwe_per_glwe * message_modulus.0.ilog2()

Reply:
Indeed